### ISSUE TITLE
ClassNotFoundException - scala.Function1 when running Java projects

### ISSUE THREAD

**Issue by carbolymer** [carymrobbins/intellij-haskforce#346]
When trying to start a run configuration on a Java project (without haskell modules) I am getting the following error:

```
Information: 05/10/17 09:39 - Compilation completed with 1 error and 0 warnings in 545ms
Error: Internal error: (java.lang.ClassNotFoundException) scala.Function1
java.lang.ClassNotFoundException: scala.Function1
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at com.haskforce.jps.model.JpsHaskellModelSerializerExtension.getProjectExtensionSerializers(JpsHaskellModelSerializerExtension.java:83)
	at org.jetbrains.jps.model.serialization.JpsProjectLoader.loadFromDirectory(JpsProjectLoader.java:123)
	at org.jetbrains.jps.model.serialization.JpsProjectLoader.loadProject(JpsProjectLoader.java:101)
	at org.jetbrains.jps.model.serialization.impl.JpsSerializationManagerImpl.loadModel(JpsSerializationManagerImpl.java:41)
	at org.jetbrains.jps.cmdline.JpsModelLoaderImpl.loadModel(JpsModelLoaderImpl.java:45)
	at org.jetbrains.jps.cmdline.BuildRunner.load(BuildRunner.java:79)
	at org.jetbrains.jps.cmdline.BuildSession.runBuild(BuildSession.java:267)
	at org.jetbrains.jps.cmdline.BuildSession.run(BuildSession.java:125)
	at org.jetbrains.jps.cmdline.BuildMain$MyMessageHandler.lambda$channelRead0$0(BuildMain.java:236)
	at org.jetbrains.jps.service.impl.SharedThreadPoolImpl.lambda$executeOnPooledThread$0(SharedThreadPoolImpl.java:42)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

The workaround is to disable HaskForce.

IntelliJ IDEA 2017.2.5
Build #IU-172.4343.14, built on September 26, 2017
JRE: 1.8.0_152-release-1024-b3 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.12.13-1-ARCH

**Comment by carymrobbins** [comment]
This seems related to #342. I'm thinking that using the snapshot build should fix it - https://github.com/carymrobbins/intellij-haskforce/releases/tag/v0.3.40-SNAPSHOT-7b63749

However, I need to go ahead and cut a release for that. I'll go ahead and get that done today.

**Comment by carymrobbins** [comment]
@carbolymer I've cut a release for this, so it should be resolved as of [0.3.40](https://github.com/carymrobbins/intellij-haskforce/releases/tag/v0.3.40); let me know if you encounter this issue as of the latest release.
